File: Python-basics/Conditional-Statements-Advanced_3/Exercise/cinema_1.py
type_projection = input()
rows = int(input())
cols = int(input())

if type_projection == "Premiere":
    result = rows * cols * 12
elif type_projection == "Normal":
    result = rows * cols * 7.50
elif type_projection == "Discount":
    result = rows * cols * 5

print(f"{result:.2f} leva")

File: Javascript/JS_basics/Conditional_Statements_Advanced_3_Exercise/cinema_1.js
function cinema(input) {
    let type = input[0];
    let rows = Number(input[1]);
    let cols = Number(input[2]);
    let result;

    if (type == 'Premiere') {
        result = rows * cols * 12;
    }
    else if (type == 'Normal') {
        result = rows * cols * 7.5;
    }
    else if (type == 'Discount') {
        result = rows * cols * 5;
    }
    result = (result).toFixed(2);
    console.log(`${result} leva`)
}